api/scpca_portal/templates/readme/metadata_only.md

```diff
@@ -6,11 +6,11 @@ The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a d
 
 ## Contents
 
-{% if included_projects|length > 1 %}
+{% if projects|length > 0 %}
```

Comment:
There should always be at least one `project` in `projects`. Why change this to > 0? If there are no projects it would most likely be undefined and not 0.

Reply:
I've reverted back the change, pls see the commit at 727a3ed0e7decedf87a3c04a56f49b36dc639314.